### PR TITLE
Configuring for pure-python

### PR DESCRIPTION
**Pull request by sallner**

### CODE REVIEW

File: src/zope/browserpage/metaconfigure.py [hunk 1]
```diff
@@ -161,7 +168,9 @@ def page(_context, name, permission, for_=Interface,
                                required)
 
     _handle_for(_context, for_)
-    new_class._simple__whitelist = set(required) - set([attribute, 'browserDefault', '__call__', 'publishTraverse'])
+    new_class._simple__whitelist = set(
+        required) - set([
+            attribute, 'browserDefault', '__call__', 'publishTraverse'])
```
Comment (icemac) [inline src/zope/browserpage/metaconfigure.py]:
```suggestion
    new_class._simple__whitelist = (
        set(required)
        - set([attribute, 'browserDefault', '__call__', 'publishTraverse']))

```

I did not check whether the last line is too long.

File: src/zope/browserpage/tests/test_boundpagetemplate.py [hunk 2]
```diff
@@ -26,9 +27,9 @@ def testAttributes(self):
         self.assertEqual(C.index.filename, C.index.__func__.filename)
 
 
-
 def test_suite():
     return unittest.defaultTestLoader.loadTestsFromName(__name__)
 
+
 if __name__ == '__main__':
     unittest.TextTestRunner().run(test_suite())
```
Comment (icemac) [inline src/zope/browserpage/tests/test_boundpagetemplate.py]:
```suggestion
```

It is no longer needed.

**Review by icemac — APPROVED** [review]
LGTM besides some suggestions.

**Comment by icemac** [comment]
Drop in coverage is because of the switch to branch coverage.

**Review by dataflake — APPROVED** [review]

**Comment by icemac** [comment]
Thank you for (reviewing) this PR. 😃

**Pull request merged**